Lullabot/lb-ebook-poc
# 20 Navigation

Resolves #20 

This PR adds the hamburger nav, which contains the table of contents, allowing the user to more easily jump around the book.

To test:

- verify that nav is not present on the book's cover page
- verify that nav is present on every other page
- verify that clicking hamburger opens the menu and swaps the icon to a close icon
- verify that nav moves into the top-right of the page as the viewport narrows
- view site across viewport sizes to verify that nav functions and doesn't break design
